examples/dynamic-client/deployment_rolling_restart.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This example demonstrates the following:
+    - Creation of a k8s deployment using dynamic-client
+    - Rolling restart of the deployment (demonstrate patch/update action)
+    - Listing & deletion of the deployment
+"""
+
+
+def main():
+    from kubernetes import config, dynamic
+    from kubernetes.client import api_client
+    import datetime
+    import pytz
+
+    # Creating a dynamic client
+    client = dynamic.DynamicClient(
+        api_client.ApiClient(configuration=config.load_kube_config())
+    )
+
+    # fetching the deployment api
+    api = client.resources.get(api_version="apps/v1", kind="Deployment")
```

Comment:
any reason for using dynamic client, instead of using the typed client?

Reply:
HI @yliaog, thanks for the reveiw.

No, there is no specific reason for using `dynamic client`. I've just replicated the same example using the `typed client`. If required, I can replace this example with the one using `typed client`.

Thank you!

Reply:
ok. i think typed client would benefit from such an example too.

Reply:
Thanks @yliaog. I'll add the same example using the `typed client` too.